app/src/main/res/values-de/strings.xml

```diff
@@ -6,7 +6,7 @@
     <string name="action_settings">Einstellungen</string>
     <string name="action_logout">Abmelden</string>
     <string name="action_search">Suchen</string>
-    <string name="action_updates">Aktualisierungen</string>
+    <string name="action_updates">Updates</string>
```

Comment:
Okay, let's do "Aktualisierungen". I'm not a big fan of anglicisms myself, but in the IT world this is a very challenging attitude.